src/darwin/Framework/CHIP/templates/availability.yaml

```diff
@@ -10084,9 +10080,245 @@
 
 - release: "Future"
   versions: "future"
+  introduced:
+      clusters:
+          - MicrowaveOvenControl
+          - MicrowaveOvenMode
+          - OvenMode
+          - OvenCavityOperationalState
+      attributes:
+          MicrowaveOvenControl:
+              - CookTime
+              - MaxCookTime
+              - PowerSetting
+              - MinPower
+              - MaxPower
+              - PowerStep
+              # SupportedWatts and SelectedWattIndex are Provisional because WATTS is Provisional in Matter 1.3
```

Comment:
We're looking at Matter 1.4 now, no?  Please fix comments to reflect 1.4 status?